.github/workflows/release.yml

```diff
@@ -42,4 +45,7 @@ jobs:
           DOCKERIO_ORG: ${{ secrets.DOCKERIO_ORG }}
         run: |
           tag=$(basename $GITHUB_REF)
+          if [ $tag = "master" ]; then
```

Comment:
Need clarification on `latest` tag, we define it as latest master build, or latest stable build? I had discussion with @VaibhavPage before, he thinks it should be the latter.

Reply:
1. It's currently neither of those. It is 3 months old.
2. This is not consistent with either workflows or CD. Both of which are the tip of master.
3. That isn't it's intent: https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375